### PR TITLE
Sweets/success

### PR DESCRIPTION
**Pull request by sweetmantech**
# Success Page
- share on Lens 🌱 
- view on OpenSea ⛵ 
- listen to remix 🎧 

### Demo
https://user-images.githubusercontent.com/23249402/219488505-398a19a4-d5eb-43d8-b8f4-8dda5213afa1.mp4

Screenshot 2023-02-16 at 4 35 53 PM

### CODE REVIEW

File: components/Results/results.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
+/* eslint-disable @next/next/no-img-element */
+import { useEffect, useState } from "react"
+import { LensShareButton } from "@infinity-keys/react-lens-share-button"
+import "@infinity-keys/react-lens-share-button/dist/style.css"
+import Confetti from "react-confetti"
+import SkeletonCard from "../SkeletonCard"
+import getIpfsLink from "../../lib/getIpfsLink"
+import useWindowSize from "../../lib/useWindowSize"
+
+const Results = ({ metadata }: any) => {
```
Comment (crazzywizard) [inline components/Results/results.tsx:10]: :nit is any type necessary? it really adds zero value to readability

Comment (crazzywizard) [reply components/Results/results.tsx:10]:
```
{
image:string
name: string
animationUrl: string
tokenId: string
}
```

**Review by crazzywizard — APPROVED** [review]
LGTM!

File: components/Results/results.tsx (hunk 1 above)
Comment (sweetmantech) [reply components/Results/results.tsx:10]: yeah, type is required

**Pull request merged**